### ISSUE TITLE
Message recipient disconnected from message bus without replying

### ISSUE THREAD

**Issue by matheus1757** [flatpak/flatpak#2687]
## Linux distribution and version 
Ubuntu 18.04 LTS
## Flatpak version
1.2.2
## Description of the problem
I always get the error "Message recipient disconnected from message bus without replying" and "Failed to install org.texstudio.TeXstudio: Message recipient disconnected from message bus without replying" when trying to install TeXstudio.
## Steps to reproduce
I just put that in the terminal and hit enter  `flatpak install flathub org.texstudio.TeXstudio -yvv`

**Comment by mwleeds** [comment]
It would be better to post the full output of the command rather than just the error message

**Comment by matheus1757** [comment]
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/matheus/.local/share/flatpak
Looking for matches…
F: Fetching summary file for remote ‘flathub’
F: Fetching summary file for remote ‘flathub’
F: Transaction: install flathub:app/org.texstudio.TeXstudio/x86_64/stable[*]
F: Updating remote metadata for flathub
F: marking op install:app/org.texstudio.TeXstudio/x86_64/stable resolved to 91455e39531791b77971d6e371b50fc0c76856023c32f76d42eeec449a6252c5
F: Updating dependent runtime runtime/org.kde.Platform/x86_64/5.11
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.11[$old]
F: marking op update:runtime/org.kde.Platform/x86_64/5.11 resolved to 02ede84d3591a5ea8028204d86059bffaccb778159ec53c859bbc60d9e7025e8
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.gtk.Gtk3theme.Ambiance/x86_64/3.22[*]
F: Transaction: install/update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform.Sources/x86_64/5.11[*]
F: Transaction: install/update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform/x86_64/5.11[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.11[/en, /pt]
F: marking op install/update:runtime/org.kde.Platform.Locale/x86_64/5.11 resolved to 5c93e19ba8c37fd60d3c47b21d5f5020af83b5d8b0160e34e30597c28e16c38b
F: marking op install/update:runtime/org.kde.PlatformTheme.QGnomePlatform/x86_64/5.11 resolved to 819a74457c036b3066e49267f246dc482e8d07c72159b47166d5195dfb9a3f09
F: marking op install/update:runtime/org.kde.PlatformTheme.QGnomePlatform.Sources/x86_64/5.11 resolved to c9d00355bc73b100912e2945b0687b9705919230bbeade925eb4e6fe1f1b361e
F: marking op install/update:runtime/org.gtk.Gtk3theme.Ambiance/x86_64/3.22 resolved to 73fed99df212c179f776452d1eb4f49d9e8fd87ba6f8434816db2e9c6c92ea19
F: marking op install/update:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6 resolved to d757f762489e444551d17223590635d4f4014a36b6fecd73323bf3f95f1aa490
F: marking op install/update:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6 resolved to 82006efc71d3c9e122ee6845147920ef6fe4054d4978f8686120afc85e89f2b0

org.texstudio.TeXstudio permissions:
    ipc	network	fallback-x11	x11
    dri	file access [1]	dbus access [2]

    [1] host, xdg-config/kdeglobals:ro, xdg-run/dconf, ~/.config/dconf:ro
    [2] com.canonical.AppMenu.Registrar, com.canonical.AppMenu.Registrar.*


 1.	   	org.texstudio.TeXstudio	x86_64	stable	flathub	< 1,5 GB



Installing…                                                                   
Installing… ████████████████████ 100%F: Calling system helper: Deploy

Error: Message recipient disconnected from message bus without replying         
error: Failed to install org.texstudio.TeXstudio: Message recipient disconnected from message bus without replying

**Comment by chpio** [comment]
Im getting 

```
> flatpak --version
Flatpak 1.3.2
```

```
> flatpak update -v    
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/thomas/.local/share/flatpak
Looking for updates…
F: Fetching summary file for remote ‘flathub’
F: Transaction: update flathub:app/org.gnome.Fractal/x86_64/stable[$old]
F: Transaction: update flathub:app/org.gnome.Lollypop/x86_64/stable[$old]
F: Transaction: update flathub:app/org.videolan.VLC/x86_64/stable[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Sdk.Locale/x86_64/18.08[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Sdk/x86_64/18.08[$old]
F: Transaction: update flathub:runtime/org.gnome.Fractal.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/org.gnome.Lollypop.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.30[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.32[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.30[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.32[$old]
F: Transaction: update flathub:runtime/org.kde.Platform.Locale/x86_64/5.12[$old]
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.12[$old]
F: Transaction: update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform.Sources/x86_64/5.12[$old]
F: Transaction: update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform/x86_64/5.12[$old]
F: Transaction: update flathub:runtime/org.videolan.VLC.Locale/x86_64/stable[$old]
F: Updating remote metadata for flathub
F: marking op update:runtime/org.videolan.VLC.Locale/x86_64/stable resolved to d64d05194cf33c148d24ba1d4a15443bec179afc3f74a3d6a4c2d494494edc0f
F: marking op update:runtime/org.kde.PlatformTheme.QGnomePlatform/x86_64/5.12 resolved to 110323e87f95f12032cbf0dca4fd84a40a5ba59299c7db4f59adada3893446a2
F: marking op update:runtime/org.kde.PlatformTheme.QGnomePlatform.Sources/x86_64/5.12 resolved to ff6a09543e8b98e8cb778122e91a7e811228f7a7b92fc24b3494d043f96df62a
F: marking op update:runtime/org.kde.Platform/x86_64/5.12 resolved to 4a85fdb18b211c6ed2f4863ad0323e5c55df391e5c74d617ad712ad483d38942
F: marking op update:runtime/org.kde.Platform.Locale/x86_64/5.12 resolved to ea3849badf655a25323c55bee62fdbac1ad3c1170df57f49de55c8ef4ba12cc5
F: marking op update:runtime/org.gnome.Platform/x86_64/3.32 resolved to 7c9d5592d805d0b7c212190fd8118e4c78cb8ed8f81e687858972670daccff76
F: marking op update:runtime/org.gnome.Platform/x86_64/3.30 resolved to 5e0d61a207f9b2f1f9f469e496eccdc5a7a07bc141ca89d86fcbc62266bb8ebd
F: marking op update:runtime/org.gnome.Platform.Locale/x86_64/3.32 resolved to a5e23c2122fffd9b2ca56939fda8c98b22c0866cdaac19c780a4c8c60ee7f141
F: marking op update:runtime/org.gnome.Platform.Locale/x86_64/3.30 resolved to d3ff07ad763365686e6bee2306efc83b1f629950563ee943a34872ef21341bd1
F: marking op update:runtime/org.gnome.Lollypop.Locale/x86_64/stable resolved to b3e450d695bc585687aae89266e4b516100d832a2de7711c1db68393d81af272
F: marking op update:runtime/org.gnome.Fractal.Locale/x86_64/stable resolved to 5f4275129edeef502b152d46dda0425761eaa7c2b890ab2020717a21e74da45f
F: marking op update:runtime/org.freedesktop.Sdk/x86_64/18.08 resolved to 883db2614bbaf1b231d90d5ed1ea1cd01d5eda9af5005e485eeb30a16e38d19f
F: marking op update:runtime/org.freedesktop.Sdk.Locale/x86_64/18.08 resolved to 1f237647ccc20414d38d42416c88c8a658c4eded8c8bc3676c8ad6d3d4f59af6
F: marking op update:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08 resolved to 282fd2c4ef33314a94ed5dd6aaaa52ce7ac8ea6f8ad920851568f7313c801b5b
F: marking op update:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08 resolved to cee4906586f7ce93faf0d93b06d3190f4e02347d363f568c67e268046b2506e2
F: marking op update:app/org.videolan.VLC/x86_64/stable resolved to 8f0c0e0bd79bfeb8748bb53e8476ee1f2deffc0e131ea5fb2890505e449df8a2
F: marking op update:app/org.gnome.Lollypop/x86_64/stable resolved to 11d00128b841e6f6253860c17d8bdf31d9bc7769d87c33e2965e8e10791fbd5c
F: marking op update:app/org.gnome.Fractal/x86_64/stable resolved to 304d40e4ef44eae3bfd1ba3b681f690fb59718c689cfd553b12eade3805bb941
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform.Sources/x86_64/5.12[*]
F: Transaction: install/update flathub:runtime/org.kde.PlatformTheme.QGnomePlatform/x86_64/5.12[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.12[/de, /en]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.32[/de, /en]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.30[/de, /en]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Sdk.Locale/x86_64/18.08[/de, /en]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.html5-codecs/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/18.08[*]
F: Transaction: install/update flathub:runtime/org.videolan.VLC.Locale/x86_64/stable[/de, /en]
F: Transaction: install/update flathub:runtime/org.gnome.Lollypop.Locale/x86_64/stable[/de, /en]
F: Transaction: install/update flathub:runtime/org.gnome.Fractal.Locale/x86_64/stable[/de, /en]


 1.	   	org.gnome.Platform	x86_64	3.32	flathub	< 371,1 MB
 2.	   	org.gnome.Platform.Locale	x86_64	3.32	flathub	< 319,8 MB (partial)
 3.	   	org.gnome.Lollypop	x86_64	stable	flathub	< 16,1 MB
 4.	   	org.gnome.Lollypop.Locale	x86_64	stable	flathub	< 481,3 kB (partial)

Proceed with these changes to the Default system installation? [Y/n]: 
Updating 1/4…                                                                                                    F: Calling system helper: GetRevokefsFd
F: Calling system helper: GetRevokefsFd
Updating 1/4… ████████████████████ 100%F: Calling system helper: Deploy
Error: Message recipient disconnected from message bus without replying                                            
Info: org.gnome.Platform.Locale was skipped                                                                        
Updating 2/4…                                                                                                    F: Calling system helper: GetRevokefsFd
F: Calling system helper: GetRevokefsFd
Updating 2/4… ████████████████████ 100%F: Calling system helper: Deploy
Error: Message recipient disconnected from message bus without replying                                            
Info: org.gnome.Lollypop.Locale was skipped                                                                        
Updates complete.                                                                                                  
error: There were one or more error
```

**Comment by mwleeds** [comment]
Could anyone who can reproduce this get a log from the system helper? To do that, run `sudo /usr/lib/flatpak/flatpak-system-helper -v -r --no-idle-exit` (or it might be in `/usr/libexec/`) and then reproduce the error. If it crashes you could instead run `sudo gdb --args /usr/lib/flatpak-system-helper -v -r --no-idle-exit` and run the command `thread apply all bt` when it crashes to get a stacktrace (and post that here).

**Comment by matthiasclasen** [comment]
I think this is already fixed: https://github.com/flatpak/flatpak/commit/00dfce528326476aa3d9167887b2cefdaec412bd

**Comment by mwleeds** [comment]
> I think this is already fixed: [00dfce5](https://github.com/flatpak/flatpak/commit/00dfce528326476aa3d9167887b2cefdaec412bd)

I don't think so. This bug report is from February and the crash you fixed was introduced in April here: https://github.com/flatpak/flatpak/commit/7185a00c8523b0368db6e40a44dc6c95d352871a

**Comment by chpio** [comment]
> I think this is already fixed: [00dfce5](https://github.com/flatpak/flatpak/commit/00dfce528326476aa3d9167887b2cefdaec412bd)

it seem to work for me now
